ocs_ci/ocs/resources/test_packagemanifest.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf8 -*-
+
+import pytest
+
+from ocs_ci.ocs.resources.packagemanifest import PackageManifest
+from ocs_ci.ocs.exceptions import ResourceNameNotSpecifiedException
+
+
+def test_pm_null():
+    pm = PackageManifest()
+    assert pm is not None
```

Comment:
Right, this test is here mainly to make sure object instantiation doesn't blow up because of exception. I'm going to remove the extraneous assert.

I was concerned with the use case from  #1338 only, but because there are no unit tests at all, I need to cover also some important simpler use cases to make some assumptions about the behavior of PackageManifest, so that it's possible to reason about potential unit test failure here.

Reply:
Addressed in 75d8fa0c3fe14cbd0fbb10d01c312f95d22e216a

Reply:
Ah, I see that I could included this here to avoid: `F841 local variable 'pm' is assigned to but never used` error.